go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/ilgooz/mattermost-plugin-marketplace-addon
+
+go 1.12
+
+require (
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/ilgooz/mattermost-dlock v0.0.0-20191107223609-c278c8c31ad6
+	github.com/mattermost/mattermost-server v0.0.0-20191107143132-540cfb0239df
+	github.com/pkg/errors v0.8.1
+	github.com/stretchr/testify v1.4.0
+)
+
+replace github.com/ilgooz/mattermost-dlock v0.0.0-20191107223609-c278c8c31ad6 => ../mattermost-dlock
+
+replace github.com/mattermost/mattermost-server v0.0.0-20191107143132-540cfb0239df => ../../mattermost/mattermost-server
```

Comment:
The Make error is stated above is because of these line. We are getting following errors 
```# github.com/ilgooz/mattermost-plugin-marketplace-addon/server/x/xtime
server/x/xtime/duration_test.go:13:3: struct field tag `json:t"` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
# github.com/ilgooz/mattermost-plugin-marketplace-addon/server/updater
server/updater/updater.go:202:63: loop variable updateOp captured by func literal
server/updater/updater.go:203:5: loop variable updateOp captured by func literal
server/updater/updater.go:203:33: loop variable updateOp captured by func literal
server/updater/updater.go:204:13: loop variable updateOp captured by func literal
server/updater/updater.go:205:45: loop variable updateOp captured by func literal
make: *** [govet] Error 2```
while running make though.

Reply:
Having a simple pipeline can be a basic safety net for such issues. Wdyt @ilgooz ?

Reply:
Yes, sorry, I couldn't have a chance to fully look at your reviews. I'm a bit busy with non-Mattermost stuff. Thanks for the reviews, they seems pretty good!

Can you please use [this version](https://github.com/mattermost/mattermost-server/pull/13041) of mattermost-server and latest version (master) of `dlock` pelase? It should do the trick!

Reply:
> Having a simple pipeline can be a basic safety net for such issues. Wdyt @ilgooz ?

CI? Sure, I'll set it up in this PR.

Reply:
No problem. Please take time. The above errors are after running go vet.

Reply:
go vet is fixed! 👍

Reply:
CI is also added.